tool_collections/taxonomy/gi2taxonomy/gi2taxonomy.xml

```diff
@@ -3,6 +3,9 @@
     <requirements>
         <requirement type="package" version="1.0.0">taxonomy</requirement>
     </requirements>
+  <xrefs>
+    <xref type="bio.tools">gi2taxonomy</xref>
+  </xrefs>
```

Comment:
I think this tool should be moved to `deprecated` instead, it doesn't seem to be installed on big 3 servers and its requirement is not installable any more (no conda package, sources on BitBucket removed).